plan/refiner.go

```diff
@@ -64,6 +64,23 @@ func buildIndexRange(sc *variable.StatementContext, p *PhysicalIndexScan) error
 			refineRange(p.Ranges[i], p.Index)
 		}
 	}
+
+	if p.accessInAndEqCount+1 < len(p.Index.Columns) {
```

Comment:
Can we just check if the length of high value or low value of each range has the same length as index columns, then append nil or max value?